samigo/samigo-hibernate/src/java/org/sakaiproject/tool/assessment/data/dao/assessment/PublishedItemData.hbm.xml

```diff
@@ -37,6 +37,7 @@
     <property name="hash" type="string" column="HASH" not-null="false" />
     <property name="itemHash" type="string" column="ITEMHASH" not-null="false" />
     <property name="isExtraCredit" type="boolean" column="ISEXTRACREDIT" not-null="false" />
+    <property name="cancellation" type="integer"><column name="CANCELLATION" default="0" not-null="true"/></property>
```

Comment:
Should be <column="CANCELLATION" instead of <column name="CANCELLATION"?

Reply:
I specified it this way, to be able to set the default. I think I can't pass this on `<property>`. I will check again.

Reply:
Though I can't set the default on `property`, it seems there is no problem removing the default value, I am updating to the format you suggested.